contracts/Core/StakeManager.sol

```diff
@@ -18,8 +18,10 @@ import "../StakedToken.sol";
 contract StakeManager is Initializable, ACL, StakeStorage, Pause {
     IParameters public parameters;
     IRewardManager public rewardManager;
-    RAZOR public razor;
     IVoteManager public voteManager;
+    IERC20 public razor;
```

Comment:
razor can directly use https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/IERC20.sol

Reply:
yes, we can import two interfaces in stakeManager, one for RAZOR and one for StakedToken.
It would be much cleaner
But it will accumulate storage for both in StakeManager.
Now its our call 
Let me know what should we do

Reply:
its fine. its a onetime cost during deployment.

Reply:
hmm yes, will import 2 interfaces

Reply:
done 2f9cac0e9bfe9ef304fa27674892f2a2d5d82606